packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -257,6 +257,13 @@ class WebViewController {
   Future<void> _updateSettings(Map<String, dynamic> update) async {
     return _channel.invokeMethod('updateSettings', update);
   }
+
+  /// Evaluate JavaScript string
+  ///
+  /// Certain functionalities directly using JavaScript are blocked from original platform. (e.g. Showing a popup Alert using window.alert())
```

Comment:
Are JavaScript alerts blocked by the platform, or is it just a matter of implementing the right callbacks? ([onJsAlert](https://developer.android.com/reference/android/webkit/WebChromeClient.html#onJsAlert(android.webkit.WebView,%20java.lang.String,%20java.lang.String,%20android.webkit.JsResult)) [runJavaScriptAlertPanelWithMessage](https://developer.apple.com/documentation/webkit/wkuidelegate/1537406-webview?language=objc)?)